src/audio/Kconfig

```diff
@@ -725,4 +725,10 @@ config COMP_MODULE_SHARED_LIBRARY_BUILD
 	  Select if you want to build modules as shared objects that can be used to run
 	  pipelines on the host with the testbench or the ALSA plugin.
 
+config DISABLE_DESCRIPTOR_SPLIT
```

Comment:
unless there's a really good reason, it's usually easier to work with "positive" options, i.e. if you had `ENABLE_DESCRIPTOR_SPLIT` you wouldn't need to handle double negations

Reply:
YES CORRECT, but for AMD platfroms in this scenario we are disabling or bypassing discriptor split, thats why it names
as DISABLE_DESCRIPTOR_SPLIT, this is meaningful too here.